README.md

```diff
@@ -230,43 +281,83 @@ enclave [tlcc_enclave](tlcc_enclave) can be build manually.
 Follow the instructions in the corresponding directories.
 
 For the integration of the enclave code into a Fabric
-chaincode, please follow the instructions in [ecc](ecc) for the chaincode
-enclave and [tlcc](tlcc) for the ledger enclave.
+chaincode, please follow the instructions in [ecc/](ecc) for the chaincode
+enclave and [tlcc/](tlcc) for the ledger enclave.
 
 In order to run and deploy a chaincode enclave we need to build the enclave
-registry. See [ercc](ercc).
+registry. See [ercc/](ercc).
+
+Moreover, we provide a set of integration tests in [integration/](integration) to demonstrate Fabric Private Chaincode
+capabilities.
+
+
+#### Trouble shooting
+
+Building the project requires docker. We do not recommend to run `sudo make`
+to resolve issues with mis-configured docker environments as this also changes your `$GOPATH`.
+
+
+
+## Your first private chaincode
+
+Hello world example tutorial comming soon here ...
+
+Let's start a looking in `examples/` ...
+
+Explain how to use integration test framework to run our own chaincode ...
+
+`fabric/bin/peer` should be used with FPC ... explain ...
+
+## Documentation
 
-Now we have all components we need to run the example auction chaincode in an enclave.
+To build documentation, you will have to install `java` and download `plantuml.jar`. Either put `plantuml.jar` into
+in your `CLASSPATH` environment variable or override `PLANTUML_JAR` or `PLANTUML_CMD` in `config.override.mk`
+(see `config.mk` for default definition of the two variables). Additionally, you will need the `dot` program from the
+graphviz package (e.g., via `apt-get install graphviz` on ubuntu).
+
+By running the following command you can generate the documentation.
+
+    $ cd docs
+    $ make
 
 
 ## Run the example chaincode
 
 Once you have successfully built the project you can play with the auction chaincode.
 Just follow the description in [fabric](fabric#).
 
+
 # References
 
 - Marcus Brandenburger, Christian Cachin, Rüdiger Kapitza, Alessandro
   Sorniotti: Blockchain and Trusted Computing: Problems, Pitfalls, and a
   Solution for Hyperledger Fabric. https://arxiv.org/abs/1805.08541
 
+
```

Comment:
Should we mention somewhere the rocketchat channel? Maybe even advertise the Tuesday calls?

Reply:
this is a great idea. I added a section for that!